### PR TITLE
Entropy rate

### PR DESCRIPTION
**Pull request by jakehanson**
Wrapped Entropy Rate and Wrote Test Cases.

### CODE REVIEW

File: test/test_entropyrate.py [hunk 1]
```diff
@@ -1,48 +1,62 @@
...
+    def testHistoryTooLongBase4(self):
+        series = numpy.random.randint(4,size = 30)
         with self.assertRaises(ValueError):
-            activeinfo([0,1,1,0,0,1,0], 0)
+            entropyrate(series,13,4)
 
-    def testEncodingError(self):
-        series = [2,1,0,0,1,0,0,1]
-        activeinfo(series, 2, 3)
+class TimeSeriesTooShort(unittest.TestCase):
```
Comment (dglmoore) [inline test/test_entropyrate.py]: You might want to make this name more specific, e.g. `EntropyRateTimeSeriesTooShort`. The problem is that if you have a `TimeSeriesTooShort` class in a different file testing, say Transfer Entropy, then these names would clash and the tests would probably break.

**Comment by dglmoore** [comment]
@jakehanson Other than the above comment everything looks great. Let me know if you plan on making that change. I'll merge the pull request once I hear something.

**Comment by jakehanson** [comment]
Hey Doug, I'm happy to make the change early next week. Also, if it is an
easy fix feel free to modify anything you like.

Thanks,
Jake

**Comment by dglmoore** [comment]
@jakehanson When you get a chance, can you make the above change? When you do I'll merge the commit.

**Comment by jakehanson** [comment]
Will do, as soon as possible.

Thanks!

**Pull request merged**